README.md

```diff
@@ -132,7 +132,14 @@ Arrowheads inherit all options from [L.Path](https://leafletjs.com/reference-1.6
       <td> proportionalToTotal </td>
       <td> Boolean </td>
       <td> false </td>
-      <td> Only relevant when <code>size</code> is given as a percent. Useful when <code>frequency</code> is set to <code>'endonly'</code>.  Will render the arrowhead(s) with a size proportional to the entire length of the multi-segmented polyline, rather than proportional to the average length of all the segments.</td>
+      <td> Only relevant when <code>size</code> is given as a percent. Useful when <code>frequency</code> is set to <code>'endonly'</code>.  Will render the arrowhead(s) with a size proportional to the entire length of the multi-segmented polyline, rather than proportional to the average length of all the segments. </td>
+   </tr>
+
+   <tr>
+      <td> fill </td>
+      <td> Boolean </td>
+      <td> false </td>
+      <td> Whether to fill the arrowhead with color. </td>
```

Comment:
Not sure why we're adding these.  I already specified that arrowheads inherit all options from L.Path, which includes fill.  The few options listed are the options that are available _in addition_ to L.Path options.